Multisig/UI/Settings/OwnerKeyManagement/AddKeyAsNewOwner/AddOwnerFlowFromSettings.swift

```diff
@@ -74,20 +74,27 @@ class AddOwnerFlowFromSettings: AddOwnerFlow {
         confirmations(stepNumber: 2, maxSteps: 3)
     }
 
+    override func review(stepNumber: Int, maxSteps: Int) {
+        assert(newConfirmations != nil)
+        assert(newOwner != nil)
+        let reviewVC = addOwnerFactory.review(
+                safe: safe,
+                newOwner: newOwner!,
+                newThreshold: newConfirmations!,
+                stepNumber: stepNumber,
+                maxSteps: maxSteps,
+                newAddressName: newAddressName
+        ) { [unowned self] txDetails in
+            transaction = txDetails
+            success()
+        }
+        show(reviewVC)
+    }
+
     override func success() {
         assert(transaction != nil)
+        // TODO remove this here?
         AddressBookEntry.addOrUpdate(newOwner!.checksummed, chain: safe.chain!, name: newAddressName!)
```

Comment:
is this intended? the other comment was saying you removed the comment...

Reply:
ups